docker-compose.yml

```diff
@@ -1,30 +1,30 @@
-version: "3.8"
-
 services:
   client:
     container_name: client_container
     build:
       context: ./client
       dockerfile: Dockerfile
     ports:
-      - "4200:4200"
-    command: ["npm", "run", "dev"]
+      - "5173:5173"
+    command: [ "npm", "run", "dev" ]
+    volumes:
+      - ./client:/usr/src/client
+      - /usr/src/client/node_modules
```

Comment:
volume mount client code

Reply:
Are we trying to avoid [this](https://stackoverflow.com/questions/30043872/docker-compose-node-modules-not-present-in-a-volume-after-npm-install-succeeds/32785014#32785014) issue by mounting an anonymous volume with `node_modules`?

Reply:
Exactly, without mounting `/usr/src/client/node_modules` it would overwrite the node modules we installed inside the Dockerfile for client.